src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -4,13 +4,12 @@
 //   Hannah Bast <bast@cs.uni-freiburg.de>
 //   2022 Julian Mundhahs <mundhahj@tf.uni-freiburg.de>
 
-#include "SparqlQleverVisitor.h"
+#include <parser/SparqlParser.h>
+#include <parser/sparqlParser/SparqlQleverVisitor.h>
 
 #include <string>
 #include <vector>
```

Comment:
```suggestion
#include <parser/sparqlParser/SparqlQleverVisitor.h>

#include <string>
#include <vector>

#include <parser/SparqlParser.h>
```

1. Associated header (x.h in the x.cpp file)
2. C stdlib headers
3. C++ stdlib  headers
4.  Third-Party headers (absl, stxxl, gtest, etc)
5.  Qlever headers.

Leave a blank line after each section, then clang-format will respect the block boundaries.

Reply:
Should probably also be on the Style Guide page of the Wiki.

Reply:
clang-format only respects this if the include of the QLever header is a relative import. When it is an absolute import (`#include <...>`) they are ordered by clang-format in the way I commited them.